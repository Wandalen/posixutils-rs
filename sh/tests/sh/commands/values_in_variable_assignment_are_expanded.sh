HOME=home

a=~
echo $a
b=$a
echo $b
c=$(echo value)
echo $c
d=$((1 * 2))
echo $d
e='quoted string'
echo $e
