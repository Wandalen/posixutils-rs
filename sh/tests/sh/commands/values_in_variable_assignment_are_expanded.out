home
home
value
2
quoted string
